1.8/administration/overlay-networks/isolation.md

```diff
@@ -0,0 +1,73 @@
+
+    $ iptables -I FORWARD -m set --match-set overlays src -m set --match-set overlays dst -j dcos-isolation
+
+This rule says that if a given packet is from any of the overlays and is destined to any other overlay, send it to the `dcos-isolation` rule. In most environments, the system should prevent an overlay network's outbound packets from re-entering the same overlay network. In order to avoid this, add an exception set of type `hash:net,net` and add entries for networks that should not be filtered. Modify the rule to: 
+
+    $ iptables -I FORWARD -m set --match-set overlays src -m set --match-set overlays dst -m set ! --match-set src,dst overlay-exceptions -j dcos-isolation
+
+The actual iptables rules that live on the `dcos-isolation` chain are just simple rules, and we recommend that you again use ipsets of type `hash:net` and refer to `src` sets and `dest` sets. The purpose of this is primarily organization.
+
+**Note:** In future versions of DC/OS we may provide support to automatically create the overlay ipsets for you. Network names prefixed with `dcos-` and `mesos-` are therefore reserved and should not be used.
+
+# Example
+
+Let’s say we have created two overlay networks: `IT` and `HR`. We want HR apps to be able to talk to IT apps, but no IT app should ever be allowed to connect to an HR app. We also want to allow all IT apps to talk to amongst themselves and all HR apps to talk amongst themselves. IT only runs apps on port 80. If we have assigned HR an overlay with the agent subnets carved from `192.168.0.0/16`, and the IT subnet out of `10.150.0.0/16`, we would use the following configuration:
+
+    $ iptables -N dcos-isolation
+    $ iptables -A dcos-isolation -j REJECT # Changes it to default drop
+    $ ipset create it hash:net
+    $ ipset create hr hash:net
+    $ ipset create overlays list:set
+
+Next, define the subnets and policies:
+
+    $ ipset add it 10.250.0.0/16
+    $ ipset add hr 192.168.0.0/16
+    $ ipset create simple_allowed hash:net,net
+    $ ipset create complex_allowed hash:net,port,net 
+    $ iptables -I FORWARD -m set --match-set overlays src -m set --match-set overlays dst -j dcos-isolation 
+    $ iptables -A dcos-isolation -m set --match-set simple_allowed src,dst -j RETURN
+
+Now we want to allow traffic going from HR and allow bidirectional connections:
+
+    $ iptables -A dcos-isolation -m set --match-set complex_allowed src,dst,dst -j RETURN
+    $ iptables -A dcos-isolation -m conntrack --ctstate RELATED,ESTABLISHED -j RETURN
+
+As well as our hairpin exception rules:
+
+    $ iptables -I dcos-isolation -m set --match-set it src -m set --match-set it dst -j RETURN
+    $ iptables -I dcos-isolation -m set --match-set hr src -m set --match-set hr dst -j RETURN
+    $ ipset add simple_allowed 192.168.0.0./16,192.168.0.0./16
+    $ ipset add simple_allowed 10.250.0.0/16,10.250.0.0/16
+    $ ipset add complex_allowed 192.168.0.0/16,80,10.250.0.0/16 #this allows traffic
```

Comment:
Yes... it looks like that's the rule that allows traffic from HR to IT on port 80?